### PR TITLE
Tomcat8

### PR DESCRIPTION
**Pull request by raghavagrawal**
Server link has been updated, rest all is working as expected.

### CODE REVIEW

**Comment by prasadtalasila** [comment]
@parv and @kapilkalra04 please verify the installation setup by cloning tomcat8 branch of @raghavagrawal and installing on a clean virtual machine. If one of you can do a screen grab of the installation process inside VM, that would serve as help guide for new users.

**Comment by prasadtalasila** [comment]
@parv and @kapilkalra04 any updates on this PR?

**Comment by kapilkalra04** [comment]
I seem to be getting this error:

**Comment by raghavagrawal** [comment]
@prasadtalasila  I have done changes. Installation still has issues when institute's internet.

File: scripts/install.sh [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-PATH=${JAVA_HOME}/bin:$PATH
-export PATH
-
-printf 'Y\n' | apt-get install -y maven
-printf 'Y\n' | apt-get install git
-
-sudo apt-get install curl
+
+#install java
+sudo bash scripts/java_install.sh
```
Comment (prasadtalasila) [inline scripts/install.sh:4]: check for JAVA environment and bail out if it does not exist.

**Review by prasadtalasila — CHANGES_REQUESTED** [review]

**Comment by raghavagrawal** [comment]
@prasadtalasila I have added the check for tomcat installation.

File: scripts/install.sh [hunk 2]
```diff
@@ -6,14 +6,17 @@ sudo bash scripts/java_install.sh
...
 #install maven
-#curl -O http://www-us.apache.org/dist/maven/maven-3/3.5.2/binaries/apache-maven-3.5.2-bin.tar.gz
-#sudo tar -xvzf apache-maven-3.5.2-bin.tar.gz
-#sudo mv apache-maven-3.5.2 /opt/maven 
-#export M2_HOME=/opt/maven
-#export PATH=${M2_HOME}/bin:${PATH}
+curl -O http://www-us.apache.org/dist/maven/maven-3/3.5.2/binaries/apache-maven-3.5.2-bin.tar.gz
+sudo tar -xvzf apache-maven-3.5.2-bin.tar.gz
+sudo mv apache-maven-3.5.2 /opt/maven 
+export M2_HOME=/opt/maven
```
Comment (prasadtalasila) [inline scripts/install.sh]: @raghavagrawal These exports are going to be effective only during the vagrant box installation. They will not be available for vagrant user after vagrant ssh login.

You need to put these exports of environment variables in `/home/vagrant/.bashrc` file. An example would be as follows.

echo 'export M2=$M2_HOME/bin' >> /home/vagrant/.bashrc

**Review by prasadtalasila — CHANGES_REQUESTED** [review]

**Comment by prasadtalasila** [comment]
The changes look good to me. Since this is an installation PR, two people need to install, use and certify the installation. @kapilkalra04 and @ParvDani07 can do that.

**Comment by ghost** [comment]
I don't know why I was mentioned earlier (unless I stumbled here in slumber). Would you please not mention me here? Thanks much.

**Pull request merged**